README.md

```diff
@@ -1 +1,188 @@
+
+<details>
+  <summary>FrameWork</summary>
+  <ul>
+    <li><a href="">ROR</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>DataBase</summary>
+  <ul>
+    <li><a href="">Postgres</a></li>
+  </ul>
+</details>
+
+### Live Demo
+
+- **https://crib-zoil.onrender.com/**
+
+### Loom Presentation
+
+**https://www.loom.com/share/35e6a6ca63fc439fbaa792034e642ff4**
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **log in**
+- **Sign in**
+- **A welcome page**
+- **Display user and Categories**
+- **Add Transaction**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby installed in your machine
+- Ruby on Rails installed in your machine
+- Postgres installed and all password and user being setup
+- A code editor e.g Vs Code
+- Any integrated terminal e.g powershell
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+cd my-folder
+git clone https://github.com/David-Lanzz/Budget-app.git</li>
+```
+
+Don't forget to change UserName and Password with yours in [database.yml](./config/database.yml) file
+
+### Install
+
+Install this project with:
+
+```
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command :
+
+```
+rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rspec spec
+rubocop . //=> for ruby file linting
+npx stylelint "**/*.{css,scss}" //=> for CSS or SASS file linting
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **David-Lanzz**
+- GitHub: [@githubhandle](https://github.com/David-Lanzz/)
+- Twitter: [@twitterhandle](https://twitter.com/LanzzDavid)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/david-lanzz)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Better Styling of various webpages**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give this repo a star and reach out on our social media, and we will be happy to know about it! 😃
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank the Microverse team for assigning the tasks to me and also Gregoire Vella on Behance for providing the original design of this project 
```

Comment:
- [Optional] Hello @David-Lanzz, Well done for applying the requirement from the previous reviewer. Can you link [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) to his website as stated in the previous review? I am asking this because it shows much consideration for the author, and it is more professional.